tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -19,6 +19,12 @@ prepare: |
     echo "Given the test-snapd-event snap is installed"
     "$TESTSTOOLS"/snaps-state install-local test-snapd-event
 
+restore: |
+    echo "Removing the snap and requesting kernel events"
+    snap remove test-snapd-event
```

Comment:
So if I understand it correctly, not removing the snap and re-trirggering udev left the devices with incorrect tags?

Reply:
It is updated, retrigger is done in reset.sh